### ISSUE TITLE
manifest unknown with sha256 digest

### ISSUE THREAD

**Issue by wilsonianb** [testcontainers/testcontainers-node#273]
I get a `manifest unknown` error, when attempting to create a container with a specified sha256 digest:

```
await new GenericContainer('postgres@sha256:eecc2357d3465c7c8722949a492176f9ba5110b42689c0351a9dffd6401f4b3c')
```

```
Error: (HTTP code 404) unexpected - manifest for postgres:eecc2357d3465c7c8722949a492176f9ba5110b42689c0351a9dffd6401f4b3c not found: manifest unknown: manifest unknown
```

logs:
```
  testcontainers INFO  Pulling image: postgres:sha256:eecc2357d3465c7c8722949a492176f9ba5110b42689c0351a9dffd6401f4b3c +1ms
  modem Sending: {
  path: '/images/create?fromImage=postgres%3Asha256&tag=eecc2357d3465c7c8722949a492176f9ba5110b42689c0351a9dffd6401f4b3c',
  ...
  ```

**Comment by cristianrgreco** [comment]
Thanks @wilsonianb, fixed in [7.20.3](https://github.com/testcontainers/testcontainers-node/releases/tag/v7.20.3)

**Comment by wilsonianb** [comment]
Thanks for the quick fix.
I'm seeing the same error with the image name includes a registry.
I've opened https://github.com/testcontainers/testcontainers-node/pull/275 to fix it.
